packages/aptos-nextra-components/tsup.config.ts

```diff
@@ -0,0 +1,20 @@
+import { defineConfig } from "tsup";
+
+const env = process.env.NODE_ENV;
+
+export default defineConfig({
+  entry: ["src/index.ts"],
+  splitting: true,
+  sourcemap: env === "production", // source map is only available in prod
+  clean: true, // rimraf disr
```

Comment:
I'm assuming this isn't a proper comment?

Reply:
supposed to say `rimraf dist` instead of `disr`. I'll fix